src/es6/Entity.js

```diff
@@ -43,7 +43,9 @@ export class Entity extends EntitySirenProperties {
 	 */
 	dispose() {
 		this._subEntities.forEach(entity => dispose(entity));
-		this._listener.remove();
+		if (this._listener) {
```

Comment:
@kieranderson  Hey this being added probably means you should be using a selflessEntity instead of a full fledged entity. I am refreshing on the code and noticed this change and it a red flag for that.